### ISSUE TITLE
Need an info page describe all game mods

### ISSUE THREAD

**Issue by MassHesteria** [dashrando/dashrando.github.io#115]
On the old site, we had https://www.dashrando.net/updates which gave a full list of all game mods (soft-lock preventions, etc.) but didn't actually cover Recall specific changes. Those were on the front site. Now that we let runners pick more settings, we probably are good with our Settings documentation, but we still need a list of the standard game mods somewhere.

We probably need to discuss map changes more in the Settings page for Map Layouts. We are gonna be adding back the DASH Classic Map Layout soon too.

**Comment by kupppo** [comment]
Good call! I think we can have this on the settings page still. We can use a table of differences for each mode to describe what is different from vanilla.
<img width="661" alt="Screenshot 2023-09-19 at 1 22 02 PM" src="https://github.com/dashrando/dashrando.github.io/assets/83246203/4977d3b8-f239-4547-8021-f5be43c5c436">

I think the key difference is in the terminology.
* **Vanilla** is the same as the vanilla game.
* **Standard** is the expected result from most randomizers (such as common anti-soft lock patches)
* We can name other modes accordingly and how the differ from Vanilla and/or Standard.

**Comment by MassHesteria** [comment]
That could work for actual map edits, but I think we may need a separate page for the stuff that isn't map specific.

![image](https://github.com/dashrando/dashrando.github.io/assets/1452105/d2198605-46bd-4e36-96ca-fcb1ae3b8acb)

There's probably more that goes in that list (auto-save, new hud, updated end sequence times, etc.).

**Comment by MassHesteria** [comment]
Also, it's worth mentioning that the front page of the old DASH site had a similar table that only talked about differences from the 2017 tournament randomizer. That was because we felt like it was the best to place to start iterating so we used it as a reference point. At this stage in the game though, I think using the vanilla game as a reference is the right way to go.

For example, we had a bullet that said "Saving the animals behaves like the vanilla game". I don't think we really need to use up space on something like that because it should be a given.
